<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7SBKKJ1" failures="3" tests="24" name="My_Leads_Data.All_Fillters_To_Check" time="219.206" errors="0" timestamp="2024-09-30T16:53:41 IST" skipped="0">
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Username" time="2.713"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Password" time="3.593"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Captcha" time="3.202"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="SUBMIT_Button" time="15.213"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Click_On_MY_Leads" time="15.250"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="All_Fillters_check" time="13.285"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="CCP_Fillters_check" time="16.876">
    <failure message="Selecting  ( CCP Tie  fillter ) does not contain any data." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Selecting  ( CCP Tie  fillter ) does not contain any data.
at org.testng.Assert.fail(Assert.java:99)
at My_Leads_Data.All_Fillters_To_Check.CCP_Fillters_check(All_Fillters_To_Check.java:194)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- CCP_Fillters_check -->
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Remove_check" time="7.437"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="CCP_New_Sale_Eligible_Fillters" time="9.688"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Remove_check2" time="7.416"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="CCP_Renewal_Fillters" time="6.383"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Remove_check3" time="7.391"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="EW_Fillters" time="11.806"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Remove_check4" time="7.389"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Ew_Fillter" time="7.420"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Remove_check5" time="7.514"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Loyality_Fillters" time="16.178"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Remove_check6" time="7.400"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Customer_tag_Fillters" time="9.452">
    <failure message="Selecting  ( Customer Tag )  additional fillter  does not contain any data." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Selecting  ( Customer Tag )  additional fillter  does not contain any data.
at org.testng.Assert.fail(Assert.java:99)
at My_Leads_Data.All_Fillters_To_Check.Customer_tag_Fillters(All_Fillters_To_Check.java:519)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Customer_tag_Fillters -->
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Remove_check7" time="7.371"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="lastService_Workshop_Fillters" time="12.817"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Remove_check8" time="7.509"/>
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Service_due_Workshop_Filters" time="8.461">
    <failure message="Selecting (Service_due_Workshop) additional filter does not contain any data." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Selecting (Service_due_Workshop) additional filter does not contain any data.
at org.testng.Assert.fail(Assert.java:99)
at My_Leads_Data.All_Fillters_To_Check.Service_due_Workshop_Filters(All_Fillters_To_Check.java:634)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Service_due_Workshop_Filters -->
  <system-out/>
  <testcase classname="My_Leads_Data.All_Fillters_To_Check" name="Remove_check9" time="7.442"/>
  <system-out/>
</testsuite> <!-- My_Leads_Data.All_Fillters_To_Check -->
